xml/System.IO.Compression/BrotliStream.xml

```diff
@@ -56,7 +56,7 @@
         <Parameter Name="compressionLevel" Type="System.IO.Compression.CompressionLevel" />
       </Parameters>
       <Docs>
-        <param name="stream">The stream to compress.</param>
+        <param name="stream">The stream to which compressed data is written.</param>
```

Comment:
```suggestion
        <param name="stream">The stream to which compressed data is written or from which data to decompress is read.</param>
```

Reply:
No, the ctors with CompressionLevel parameter create compression streams. Specifying CompressionLevel implies you want to compress.